nilearn/plotting/displays/_slicers.py

```diff
@@ -372,7 +415,19 @@ def add_contours(self, img, threshold=1e-6, filled=False, **kwargs):
                     # should be given as (lower, upper).
                     levels.append(np.inf)
 
-            self._map_show(img, type="contourf", threshold=threshold, **kwargs)
+            ims = self._map_show(
+                img, type="contourf", threshold=threshold, **kwargs
+            )
+
+        if colorbar and ims:
+            if "levels" in kwargs or "colors" in kwargs:
+                raise ValueError(
```

Comment:
I would make this a warning and not an error, no?
Also we need to add a test for this.